frontend/src/main/components/Announcement/AnnouncementForm.js

```diff
@@ -1,41 +1,61 @@
 import { Button, Form } from 'react-bootstrap';
-import { useForm } from 'react-hook-form'
-import { useNavigate } from 'react-router-dom'
+import { useForm } from 'react-hook-form';
+import { useNavigate } from 'react-router-dom';
 
 function AnnouncementForm({ initialContents, submitAction, buttonLabel = "Create" }) {
-
     // Stryker disable all
     const {
         register,
         formState: { errors },
         handleSubmit,
-    } = useForm(
-        { defaultValues: initialContents || {}, }
-    );
-    // Stryker restore all
+    } = useForm({
+        defaultValues: {
+            startDate: initialContents?.startDate 
+                ? initialContents.startDate.split("T")[0]
+                : getPSTDate(),
+            endDate: initialContents?.endDate || null,
+            announcementText: initialContents?.announcementText || "",
+            ...initialContents, 
+        },
+    });
```

Comment:
```suggestion
    });
    // Stryker restore all
```

You can't just turn off Stryker whenever you feel like it.  There are only very narrow cases where we are permitted Stryker exceptions, and even some of those should actually be removed from the code base as we know *do* know how to test those.  The number of Stryker exceptions needs to be going *down* not *up*.